subsys/random/rand32_ctr_drbg.c

```diff
@@ -69,7 +69,7 @@ static int ctr_drbg_initialize(void)
 
 	ret = mbedtls_ctr_drbg_seed(&ctr_ctx,
 				    ctr_drbg_entropy_func,
-				    entropy_driver,
+				    (void *)entropy_driver,
```

Comment:
We loose the const qualifier then. Since entropy_driver is static to the whole file, just pass NULL here, and use entropry_driver instead of ctx in ctr_drbg_entropy_func

Reply:
okay